subprojects/architecture-test/src/changes/archunit_store/internal-api-nullability.txt

```diff
@@ -4899,8 +4899,6 @@ Class <org.gradle.process.internal.worker.DefaultWorkerLoggingProtocol> is not a
 Class <org.gradle.process.internal.worker.DefaultWorkerProcess$1> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (DefaultWorkerProcess.java:0)
 Class <org.gradle.process.internal.worker.DefaultWorkerProcess$2> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (DefaultWorkerProcess.java:0)
 Class <org.gradle.process.internal.worker.DefaultWorkerProcess> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (DefaultWorkerProcess.java:0)
-Class <org.gradle.process.internal.worker.DefaultWorkerProcessBuilder$1$1> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (DefaultWorkerProcessBuilder.java:0)
-Class <org.gradle.process.internal.worker.DefaultWorkerProcessBuilder$1> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (DefaultWorkerProcessBuilder.java:0)
```

Comment:
These disappeared, as I've turned the respective code using lambdas instead of anonymous classes.